save-cloud-charts/save-cloud/values.yaml

```diff
@@ -89,7 +89,7 @@ grafana:
   enabled: false
   image:
     repository: grafana/grafana
-    tag: 9.4.3
+    tag: 9.3.6
```

Comment:
I think these versions can be kept as is, because they shouldn't be that dependant on charts